ports/servoshell/desktop/app_state.rs

```diff
@@ -477,9 +478,6 @@ impl WebViewDelegate for RunningAppState {
             .build();
 
         webview.notify_theme_change(self.inner().window.theme());
-        webview.focus();
-        webview.raise_to_top(true);
```

Comment:
Why are you removing the focus step here?

Reply:
> Unify the behavior, as said in PR message. Also required by webdriver.

I understand that you have made the behavior the same as the embedded version of servoshell, but you also need to justify in the PR description why this change is okay to make for the desktop port. Was it unnecessary before?

Reply:
> For the normal user interaction to open new tab, the behaviour won't change, as it is handled in another part.

Triggering `window.open()` is also normal user behavior here, I would say.